components/brave_rewards/browser/rewards_service.h

```diff
@@ -19,6 +19,11 @@ namespace content {
 class NavigationHandle;
 }
 
+namespace ledger {
```

Comment:
we specifically don't want to expose `ledger` in rewards_service.h, these methods only need to exist on RewardsServiceImpl
RewardsService is the public API for brave-core